src/guide/send_payment.md

```diff
@@ -7,20 +7,26 @@ Once you have outbound liquidity you can start sending payments too.
 <section>
 
 ```rust,ignore
-let bolt11 = "...";
-sdk.send_payment(bolt11.into(), None).await?;
+// The `amount_msat` param is optional and should only passed if the bolt11 doesn't specify an amount.
+// The amount_msat is required in case an amount is not specified in the bolt11 invoice'.
+let amount_msat: Option<u64> = None;
+let req = SendPaymentRequest {
+    bolt11: "...".into(),
+    amount_msat,
+};
+let response = self.send_payment(req).await?;
 ```
 </section>
 
 <div slot="title">Swift</div>
 <section>
 
 ```swift
-let bolt11 = "...";
 do {
-  // The `amountSats` param is optional so nil can be passed if the 
-  // bolt11 invoice spesifies an amount.
-  let payment = try sdk.sendPayment(bolt11: bolt11, amountSats: 3000)
+  // The `amountMsat` param is optional and should only passed if the bolt11 doesn't specify an amount.
+  // The amountMsat is required in case an amount is not specified in the bolt11 invoice'.
+  let req = SendPaymentRequest(bolt11: "...", amountMsat: 3000000)
+  let response = try sdk.sendSpontaneousPayment(req: req)
```

Comment:
Change to `sendPayment`

Reply:
Addressed 057702b7616d3c5cf7d769031e0044aee16f71ce